src/main/java/com/zeiterfassung/hwr/desktop/controller/ButtonPaneController.java

```diff
@@ -72,6 +104,12 @@ private Optional<String> getOptionalLastProjectName(Optional<TimeAction> lastBoo
                 .map(Project::getName);
     }
 
+    /**
+     * get the project Names
```

Comment:
get the project Names from a list of projects OR map list of projects to the project names